spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -346,13 +355,32 @@
       end
 
       it 'logs a reproofing event upon reproofing' do
-        user.profiles.create(verified_at: Time.zone.now, active: true, activated_at: Time.zone.now)
+        unverified_profile = user.profiles.first
+        verified_profile = create(:profile, :verified, user: user)
+
         stub_sign_in(user)
         subject.session[:sp] = {
           ial2: false,
           issuer: 'foo',
           request_url: 'http://example.com',
         }
+
+        expect(unverified_profile.activated_at).to be_present
```

Comment:
if the profile is unverified should there be an activated_at?

Reply:
I agree that this is an issue to investigate if we keep this code around.